src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/TickerTask.java

```diff
@@ -333,8 +424,8 @@ public void enableTicker(@Nonnull Location l) {
     public void disableTicker(@Nonnull Location l) {
         Validate.notNull(l, "Location cannot be null!");
 
-        ChunkPosition chunk = new ChunkPosition(l.getWorld(), l.getBlockX() >> 4, l.getBlockZ() >> 4);
-        Set<Location> locations = tickingLocations.get(chunk);
+        String chunk = l.getWorld().getName() + ";" + l.getChunk().getX() + ";" + l.getChunk().getZ();
+        Set<Location> locations = activeTickers.get(chunk);
```

Comment:
:NotLikeThis:

Reply:
I don't have time to non-fatal beauty changes, feel free to do it yourself though.